### ISSUE TITLE
[Question] Color Texture Preset actually do?

### ISSUE THREAD

**Issue by PickyPixelStudio** [MehVahdJukaar/labels-multi#37]
Hey!

I tried all of the value, with and without outline, limit palette, and recolor texture.

Their is no difference when it's DEFAULT, PENCIL, PASTEL or FLAT.

What are they really doing?! Oo

**Comment by MehVahdJukaar** [comment]
did you reload texture packs? do those labels have a dye?

**Comment by PickyPixelStudio** [comment]
Yeah, i reload each time, and nothing change. I making a ressources pack, and nothing
I didn't dye the label.
At first i think it was an incompatibility with another mod, so i tried on a fresh minecraft with forge, and same problem.
